eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -80,6 +80,12 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/verify-path-length.yml
         parameters:
           SourceDirectory: $(Build.SourcesDirectory)
+      - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
If we end up calling verify during the build we probably don't need to call it a second time here.